### PR TITLE
Init readme

### PR DESCRIPTION
**Pull request by GalloisLuca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,109 +4,71 @@
...
-    ```
-    
-- Using `dev` environment:
+### Reference Entity
 
-    ```bash
-    (cd tests/Application && APP_ENV=dev bin/console sylius:fixtures:load)
-    (cd tests/Application && APP_ENV=dev bin/console server:run -d public)
-    ```
+[Everything you need to know about Reference Entity in Sylius](docs/reference_entity/REFERENCE_ENTITY.md)
```
Comment (oallain) [inline README.md]: IMO, we need to remove this line ;)

File: README.md [hunk 2]
```diff
@@ -4,109 +4,71 @@
...
-<h1 align="center">Plugin Skeleton</h1>
+## Features
 
-<p align="center">Skeleton for starting Sylius plugins.</p>
+## Requirements
 
-## Documentation
+| | Version |
+| :--- | :--- |
+| PHP  | 7.3+ |
```
Comment (oallain) [inline README.md:13]: This makes me think. Wouldn't we limit version to PHP 7.4 min (to have strong typing)?
WDYT ?

Comment (GalloisLuca) [reply README.md:13]: I agree for the strong typing, but we want the plugin to be able on sylius 1.8 too, sylius 1.8 need "php": "^7.3"... So i think we should go for PHP 7.3 + for the first release

Comment (oallain) [reply README.md:13]: I think we can do that on another PR, but yes, we need PHP 7.4 mini.

**Review by oallain — APPROVED** [review]

**Pull request merged**